Description: Avoid multiarch conflict
 As per Debian's multiarch hinter:
 libtrilinos-stokhos-dev conflicts on /usr/include/trilinos/Stokhos_config.h
 on amd64 <-> arm64, mips64el, ppc64el, s390x
 .
 Make the header file identical across all architectures
 by defining HAVE_STOKHOS_INTRINSICS on amd64 and i386
 where immintrin.h is available.
Author: Graham Inggs <ginggs@debian.org>
Last-Update: 2019-01-29

--- a/packages/stokhos/cmake/Stokhos_config.h.in
+++ b/packages/stokhos/cmake/Stokhos_config.h.in
@@ -111,7 +111,11 @@
 #cmakedefine HAVE_STOKHOS_OPENMP
 
 /* Define if intrinsics are enabled */
-#cmakedefine HAVE_STOKHOS_INTRINSICS
+#if defined(__x86_64__) || defined(__i386__)
+#define HAVE_STOKHOS_INTRINSICS
+#else
+#undef HAVE_STOKHOS_INTRINSICS
+#endif
 
 /* Define if C++11 is enabled */
 #cmakedefine HAVE_STOKHOS_CXX11
